src/main.rs

```diff
@@ -279,6 +290,10 @@ impl SeatHandler for State {
 
     fn keyboard_added(&mut self, conn: &mut Connection<Self>, seat: WlSeat) {
         self.keyboards.push(Keyboard::new(conn, seat));
+        if let Some(inhibit_manager) = &self.keyboard_shortcuts_inhibit_manager {
+            self.keyboard_shortcuts_inhibitors
+                .push(inhibit_manager.inhibit_shortcuts(conn, self.wl_surface, seat));
+        }
```

Comment:
These lines should be moved to `SeatHandler::seat_added`.

Reply:
Done.